### PR TITLE
Nutanix: fix broken rocky link

### PR DESCRIPTION
**Pull request by tuxtof**
What this PR does / why we need it:

Fix Rocky Linux broken link, switch to latest release to avoir futur issue

### CODE REVIEW

**Comment by tuxtof** [comment]
@AverageMarcus this is the PR for rocky link update

File: images/capi/packer/nutanix/rockylinux-8.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "epel_rpm_gpg_key": "https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8",
   "extra_rpms": "python3",
   "guest_os_type": "Linux",
-  "image_url": "https://dl.rockylinux.org/pub/rocky/8/images/x86_64/Rocky-8-GenericCloud-Base-8.7-20221130.0.x86_64.qcow2",
+  "image_url": "https://dl.rockylinux.org/pub/rocky/8/images/x86_64/Rocky-8-GenericCloud-Base.latest.x86_64.qcow2",
```
Comment (AverageMarcus) [inline images/capi/packer/nutanix/rockylinux-8.json:11]: It looks like the image does still exist but now available at `http://dl.rockylinux.org/vault/rocky/8.7/images/x86_64/Rocky-8-GenericCloud-Base-8.7-20221130.0.x86_64.qcow2` instead.

I'd like to get other peoples opinion before switching to using `latest` as it causes re-runs of the same configuration to potentially result in different images which I'm not too keen on. I'm going to open a thread on Slack to gather thoughts.

Comment (tuxtof) [reply images/capi/packer/nutanix/rockylinux-8.json:11]: this is already how image builder work as a

yum update / apt update is performed during each build

so each result is different with always the last security update

starting from latest allow to have the exact same result just improve the build time

but open to have other peoples opinion

Comment (AverageMarcus) [reply images/capi/packer/nutanix/rockylinux-8.json:11]: Slack discussion: https://kubernetes.slack.com/archives/C01E0Q35A8J/p1687764381019339

**Pull request merged**